### DISCUSSION TITLE
是否考虑热补丁更新的计划

### DISCUSSION THREAD

**Discussion by Nyadoki** [Meloong-Git/PCL#1201] [提问] [answered]
大版本更新实在是太慢了以及频繁
开fa组有没有考虑过 通过打热补丁来实现修复紧急的虫子

**Comment by WForst-Breeze** [comment, marked as answer]
紧急修复版本一般会在重大问题出现后一周内发布，一般情况下发现重大问题后都是会及时处理并推送更新的，以补丁形式的修复个人感觉没太大必要，但是具体是否会考虑还是得看龙猫怎么想🤔

**Reply by WorldHim** [reply]
龙猫已拒绝。

> 技术上没法实现……

[来源](https://github.com/Hex-Dragon/PCL2/issues/1207#issuecomment-1422093203_)

**Comment by Nyadoki** [comment]
没碰过vb 我只使用kotlin实现过(悲
